### PR TITLE
[Convert method refereces  mix util part]Lps 170398 part 1

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-170398

### CODE REVIEW

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
Forced pushed with updating the semantic version

File: modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/internal/display/context/SelectAssetDisplayPageDisplayContext.java [hunk 1]
```diff
@@ -297,9 +300,14 @@ public boolean inheritableDisplayPageTemplate() {
...
 		}
 
+		Bundle bundle = FrameworkUtil.getBundle(getClass());
+
+		BundleContext bundleContext = bundle.getBundleContext();
+
 		LayoutDisplayPageProviderRegistry layoutDisplayPageProviderRegistry =
-			LayoutDisplayPageProviderRegistryUtil.
-				getLayoutDisplayPageProviderRegistry();
+			bundleContext.getService(
```
Comment (dantewang) [inline modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/internal/display/context/SelectAssetDisplayPageDisplayContext.java:308]: @slnn When you get service you should always unget it.

**Comment by dantewang** [comment]
@slnn I see your ticket contains 3 modules/module groups, and you say the ticket will be done in 3 parts. Then why don't you create 3 tickets, each module/module group a ticket?

File: modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/internal/display/context/SelectAssetDisplayPageDisplayContext.java (hunk 1 above)
Comment (dantewang) [reply modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/internal/display/context/SelectAssetDisplayPageDisplayContext.java:308]: @slnn Also, please avoid using this way. Move the static field into the display context class, and use a component to inject the referenced service.

**Comment by dantewang** [comment]
@slnn Please see my comment and resend.

**Pull request closed without merging**